src/Context/UserContext/UserContext.jsx

```diff
@@ -0,0 +1,102 @@
+import { createContext, useContext, useEffect, useState } from 'react';
+
+const UserContext = createContext();
+
+const UserProvider = ({ children }) => {
+	const [user, setUser] = useState({});
+
+	const [userPosts, setUserPost] = useState([]);
+
+	useEffect(() => {
+		setUser({
+			displayName: 'Zoef Shaikh',
```

Comment:
I hope this is a temporary data placeholder and you'll be fetching from an API? or abstracting your own DB?

Reply:
Yep this is a temp data.